Composer/package.json

```diff
@@ -47,7 +47,9 @@
     "start": "cross-env NODE_ENV=production PORT=3000 yarn start:server",
     "startall": "yarn start",
     "start:dev": "concurrently  \"npm:start:client\" \"npm:start:server:dev\"",
+    "start:dev:electron": "concurrently  \"npm:start:client\" \"npm:start:electron\"",
```

Comment:
This might not work on Windows machines due to how long the client takes to start up.

Normally when I try to start the electron server without the dev server running, it throws an error and never starts. Usually the client dev server takes longer to start than the electron process, so I would assume if they are started concurrently that it would error out most of the time, but I would have to test it to make sure.

Reply:
Ok. I can update to wait for dev server to start before starting electron

Reply:
@tonyanziano Can you check out using `wait-on`?